leasing/migrations/0058_fix_paikkatietovipunen_vuokraalueet_remove_intended_use_requirement.sql

```diff
@@ -0,0 +1,151 @@
+                   FROM leasing_tenantcontact tc
+                  WHERE tc.type::text = 'contact'::text AND (tc.end_date IS NULL OR tc.end_date > now()) AND (tc.tenant_id IN ( SELECT t.id
+                           FROM leasing_tenant t
+                          WHERE t.lease_id = lease.id))))) AS yhteyshenkilot_osoitteet,
+    ( SELECT aa.address
+           FROM leasing_leaseareaaddress aa
+          WHERE aa.lease_area_id = l_area.id AND aa.is_primary
+         LIMIT 1) AS ensisijainen_osoite,
+    lease.start_date AS vuokraus_alkupvm,
+    lease.end_date AS vuokraus_loppupvm,
+    np.name AS irtisanomisaika,
+    liu.name AS vuokraus_kayttotarkoitus,
+        CASE
+            WHEN lease.end_date IS NULL THEN 'kyllä'::text
+            WHEN lease.end_date > now() THEN 'kyllä'::text
+            WHEN lease.end_date < now() THEN 'ei'::text
+            ELSE NULL::text
+        END AS vuokraus_voimassa,
+        CASE
+            WHEN lease.state::text = 'permission'::text THEN 'Lupa'::text
+            WHEN lease.state::text = 'lease'::text THEN 'Vuokraus'::text
+            WHEN lease.state::text = 'short_term_lease'::text THEN 'Lyhytaikainen vuokraus'::text
+            WHEN lease.state::text = 'long_term_lease'::text THEN 'Pitkäaikainen vuokraus'::text
+            WHEN lease.state::text = 'reservation'::text THEN 'Varaus'::text
+            WHEN lease.state::text = 'reserve'::text THEN 'Varanto'::text
+            WHEN lease.state::text = 'application'::text THEN 'Hakemus'::text
+            WHEN lease.state::text = 'rya'::text THEN 'RYA'::text
+            ELSE NULL::text
+        END AS vuokraus_tila,
+    l_area.area AS pintaala,
+    l_area.section_area AS leikkauspintaala,
+    ( SELECT sum(leasing_rent.payable_rent_amount) AS sum
+           FROM leasing_rent
+          WHERE leasing_rent.lease_id = lease.id) AS perittava_vuokra_summa,
+    ( SELECT concat(to_char(leasing_rent.payable_rent_start_date::timestamp with time zone, 'fmDD.fmMM.YYYY'::text), ' - ', to_char(leasing_rent.payable_rent_end_date::timestamp with time zone, 'fmDD.fmMM.YYYY'::text)) AS concat
+           FROM leasing_rent
+          WHERE leasing_rent.lease_id = lease.id
+          ORDER BY leasing_rent.payable_rent_start_date DESC
+         LIMIT 1) AS perittava_vuokra_jakso,
+    lease.note AS vuokraus_huom,
+    ltype.name AS vuokraus_tyyppi,
+        CASE
+            WHEN l_area.location::text = 'surface'::text THEN 'Yläpuolella'::text
+            WHEN l_area.location::text = 'underground'::text THEN 'Alapuolella'::text
+            ELSE NULL::text
+        END AS sijainti,
+    st_transform(l_area.geometry, 3879) AS geometria
+   FROM leasing_leasearea l_area
+     JOIN leasing_lease lease ON l_area.lease_id = lease.id
+     JOIN leasing_leaseidentifier lid ON l_area.lease_id = lid.id
+     JOIN leasing_leasetype ltype ON lease.type_id = ltype.id
+     LEFT JOIN leasing_intendeduse liu ON lease.intended_use_id = liu.id
```

Comment:
Just to point out the change, as it is hard to observe. This line changed from `JOIN` to `LEFT JOIN`